### PR TITLE
pre-commit autoupdate

### PR DESCRIPTION
**Pull request by hicksl**
Run pre-commit autoupdate on outer repo, copy updates to inner. Perform manual checks.

### CODE REVIEW

**Comment by hicksl** [comment]
Passing manual checks.

**Comment by hicksl** [comment]
Closes #56

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ default_language_version:
 exclude: "{{cookiecutter.repo_name}}"
```
Comment (john-sandall) [inline .pre-commit-config.yaml:6]: We can bump precommit version also to 2.19.0
https://github.com/pre-commit/pre-commit/releases

Comment (john-sandall) [reply .pre-commit-config.yaml:6]: (also need to bump in pyproject.toml)

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -59,7 +59,7 @@ repos:
     hooks:
```
Comment (john-sandall) [inline .pre-commit-config.yaml:59]:
```
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.9.2
```

in pyproject.toml we now use 4.0.1 so I'm trying that here, not sure why autoupdate didn't upgrade that though

Comment (john-sandall) [reply .pre-commit-config.yaml:59]: autoupdate won't catch the inner versions, these must be manually checked, e.g. https://pypi.org/project/flake8-eradicate/ is now at 1.2.1

**Review by john-sandall — CHANGES_REQUESTED** [review]

**Pull request merged**